lib/cli.rb

```diff
@@ -787,6 +787,25 @@ def self.supports_filtering(command)
     end
   end
 
+  if @config.experimental_features
+    desc "Dockerize a system description"
+    long_desc <<-LONGDESC
+      Detects workloads from a system description and creates docker containers.
+    LONGDESC
+    arg "NAME"
+    command :dockerize do |c|
+      c.flag ["docker-dir", :d], type: String, required: true,
+        desc: "Location where the docker files will be stored", arg_name: "DIRECTORY"
+
+      c.action do |global_options,options,args|
```

Comment:
Unused block argument - `global_options`. If it's necessary, use `_` or `_global_options` as an argument name to indicate that it won't be used.<br>Space missing after comma.